src/graphql/main/schema.graphql

```diff
@@ -938,7 +945,9 @@ Price of 1 sat in base/offset. To calculate, use: `base / 10^offset`
 """
 type PriceOfOneSat implements PriceInterface {
   base: SafeInt!
-  currencyUnit: String!
+  currencyUnit: CurrencyUnit!
+    @deprecated(reason: "Deprecated in favor of minorUnitToMajorUnitOffset")
```

Comment:
Which line is the `@deprecated` applying to? The `currencyUnit`?
Why are we changing the type if we are deprecating it?

Reply:
now `${currency}CENTS` does not make sense... the deprecation was requested in the last display currency meeting with mobile devs